skietbaan_fe/src/components/RegisterStyles.css

```diff
@@ -134,28 +132,30 @@
   top: 5px;
   right: 5px;
 }
+
 .password-view-icon:hover{
   cursor: pointer;
 }
 
 .header-label{
   font-size: 25px;
   color: white;
-  font-weight: bold;
-  padding-top: 4px;
+  font-weight: 400;
+  padding-top: 21px;
 }
 
 .button-login{
-  background-color: black;
+  background-color: #EA241A;
   color: white;
-  border: 1px red solid;
+  border: 1px white solid;
```

Comment:
above code is hex. change it to be the same

Reply:
alright fixed